torch/csrc/lazy/ts_backend/ts_node.cpp

```diff
@@ -157,5 +157,25 @@ TSOpVector TsNode::Lower(std::shared_ptr<torch::jit::GraphFunction> function,
   return {};
 }
 
+TensorList::TensorList(OpList values)
+  : TsNode(/*op=*/tensor_list_opkind,
```

Comment:
could you remind me why we don't want to use `shapes` for `TensorList`. 
Is it because the number of shapes doesn't match with the number of outputs?

Reply:
Yes, it's because the shapes don't match the # outputs.  Typically we have one shape per output.  I first implemented the shapes to be the shapes of the tensors, so if the list had 2 tensors there would be 2 shapes.

I could logically make the shapes of the TensorList be how they would be if the TensorList were fed to Stack.  This would solve the inconsistency but feels a bit arbitrary.

It's more than academic; there was an assert somewhere that the # shapes matches # outputs - so I had to do this to get off the ground.  But I'm certainly open to suggestions.

I think my grand plan is to move shape to Node as virtual and then for TensorList make it unimplemented, which would also require I avoid that assert that was failing.